Makefile

```diff
@@ -206,3 +206,7 @@ public/api/_analytics-events.json: .yardoc .yardoc/objects/root.dat
 		--no-output \
 		--db $@ \
 		-- $<
+
+update: ## Run after a git pull perhaps
+	bundle install
+	bundle exec rails db:migrate
```

Comment:
sometimes we tell people just to run `make setup` again and we also have `make fast_setup` ?

But if we want an even simpler version maybe we throw a `yarn install` in here as well? 🤷

Reply:
Huh, I hadn't seen `make fast_setup`. It doesn't seem like it has quite the same purpose, though. I wanted something simple to address the, "Whoops, I pulled the latest `main` but forgot to..." stuff that throws errors.

I like the idea of adding `yarn install` here, though.